exceptions/exceptions.hh

```diff
@@ -68,6 +68,10 @@ enum class exception_code : int32_t {
     FUNCTION_FAILURE= 0x1400,
     CDC_WRITE_FAILURE = 0x1600,
 
+    // Scylla-specific codes
+    // Allocated backwards from 0x1aff-0x1a01 to minimize the chance of collision with Cassandra.
+    OVERFLOW_ERROR  = 0x1aff,
```

Comment:
Do we have good enough relations with cassandra developers to propose reserving this code on their end? They don't have to implement the exception now, but if they ever decide to we would at least be compatible.

Reply:
I definitely don't have these relations.
We can try sending a patch, but I doubt they'll accept it with no implemented use of this value.

Reply:
They won't, they made it clear in another occasion.